packages/preview/fireside/1.0.0/lib.typ

```diff
@@ -0,0 +1,66 @@
+#let _title-size = 44pt
+
+#let fireside(
+  background: rgb("f4f1eb"),
+  title: "",
+  from-details: none,
+  to-details: none,
+  margin: 2.1cm,
+  vertical-center-level: 2,
+  body
+) = {
+  set page(fill: background, margin: margin)
+  set text(font: ("HK Grotesk", "Hanken Grotesk"))
+
+  let body = [
+    #set text(size: 11pt, weight: "medium")
+    #show par: set block(spacing: 2em)
+    #body
+  ]
+
+  let header = {
+    grid(
+      columns: (1fr, auto),
+      [
+        #set text(size: _title-size, weight: "bold")
+        #set par(leading: 0.4em)
+        #title
+      ],
+      align(end, box(
+        inset: (top: 1em),
+        [
+          #set text(size: 10.2pt, fill: rgb("4d4d4d"))
+          #from-details
+        ]
+      )),
+    )
+    v(title-size)
+    text(size: 9.2pt, to-details)
+    v(title-size)
```

Comment:
These two `title-size` should become `_title-size` too.
```suggestion
    v(_title-size)
    text(size: 9.2pt, to-details)
    v(_title-size)
```

Reply:
Oops